File: nivel-00/area-quadrada/foo.c
#include <stdio.h>
#include <assert.h>

//
// Função que retorna o valor da área
//
int area(int lado1, int lado2) {
	return lado1 * lado2;
}

//
// testes
//
int main (){
    assert(27 == area(3, 9));
    return 0;
}

File: nivel-00/area-quadrada/foo.js
var assert = require('assert');

//
// Função que retorna o valor da área
//
var area = function (lado1, lado2) {
    return lado1 * lado2;
}

//
// Seu teste
//
try {
    assert.equal(27, area(3, 9), "deve retornar o área");
} catch(e) {
    console.log(e);
}